stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/FileAttachmentsViewStyle.kt

```diff
@@ -0,0 +1,118 @@
+package io.getstream.chat.android.ui.message.list
+
+import android.content.Context
+import android.graphics.Typeface
+import android.graphics.drawable.Drawable
+import android.util.AttributeSet
+import androidx.annotation.ColorInt
+import androidx.annotation.Px
+import io.getstream.chat.android.ui.R
+import io.getstream.chat.android.ui.TransformStyle
+import io.getstream.chat.android.ui.common.extensions.internal.dpToPx
+import io.getstream.chat.android.ui.common.extensions.internal.getColorCompat
+import io.getstream.chat.android.ui.common.extensions.internal.getDrawableCompat
+import io.getstream.chat.android.ui.common.extensions.internal.use
+import io.getstream.chat.android.ui.common.style.TextStyle
+
+public data class FileAttachmentsViewStyle(
+    @ColorInt val backgroundColor: Int,
+    @ColorInt val strokeColor: Int,
+    @Px val strokeWidth: Int,
+    @Px val cornerRadius: Int,
+    val progressBarDrawable: Drawable,
+    public val actionButtonIcon: Drawable,
+    @ColorInt val actionButtonTintColor: Int,
+    public val failedAttachmentIcon: Drawable,
+    @ColorInt val failedAttachmentIconTintColor: Int,
```

Comment:
We don't want to support `IconTint` as separate attributes. It can be overridden using icon

Reply:
Fixed!